docs/_source/tutorials/notebooks/training-textclassification-activelearning-with-GPU.ipynb

```diff
@@ -0,0 +1,695 @@
+{
```

Comment:
Could you rename the tutorial to 
docs/_source/tutorials/notebooks/training-textclassification-colab-activelearning.ipynb ?

Reply:
This is according to our structure. mlops-nlp-package-technique

Reply:
do you think it would be possible to add some pictures to make it a bit more visual?🤓